src/pages/developers/learning-tools.tsx

```diff
@@ -115,23 +115,25 @@ const StackContainer = (props: ChildOnlyProp) => (
   />
 )
 
-export const getStaticProps = (async (context) => {
-  const { locale } = context
-  // load i18n required namespaces for the given page
+export const getStaticProps = (async ({ locale }) => {
   const requiredNamespaces = getRequiredNamespacesForPage(
     "/developers/learning-tools"
   )
+
+  const contentNotTranslated = !existsNamespace(locale!, requiredNamespaces[2])
```

Comment:
Would love to have a more deterministic way to get the specific namespace in future iterations.